### PR TITLE
[css-content] Fix syntaxes according to CSS Values and Units grammar

### PR DESCRIPTION
**Pull request by lahmatiy**
Fix and simplify syntaxes, see #663 for details

### CODE REVIEW

File: css-content/Overview.bs [hunk 1]
```diff
@@ -601,7 +601,8 @@ The ''leader()'' function</h4>
 	</dl>
 
 	<pre class="prod">
-		<dfn><<leader-type>></dfn> = leader( dotted | solid | space | <<string>>);
+		<dfn><<leader()>></dfn> = leader( <<leader-type>> )
```
Comment (tabatkins) [inline css-content/Overview.bs]: This should be defined as just `<dfn>leader()</dfn>`. (The grammar non-terminal automagically works when the function is defined normally.)

File: css-content/Overview.bs [hunk 2]
```diff
@@ -758,7 +759,7 @@ Cross references and the target-* functions</h3>
 	Each of these displays information obtained from the target end of a link.
 
 	<pre class="prod">
-		<dfn><<target>></dfn> = [ ''target-counter()'' | ''target-counters()'' | ''target-text()'' ]
+		<dfn><<target>></dfn> = <<target-counter()>> | <<target-counters()>> | <<target-text()>>
```
Comment (tabatkins) [inline css-content/Overview.bs]: Same deal as previous function.

File: css-content/Overview.bs [hunk 3]
```diff
@@ -769,7 +770,7 @@ Cross references and the target-* functions</h3>
 The ''target-counter()'' function</h4>
 
 	<pre class="prod">
-		<dfn>target-counter()</dfn> = target-counter( [ <<string>> | <<url>> ] , <<custom-ident>> [ , <<counter-style>> ]? )
+		<dfn><<target-counter()>></dfn> = target-counter( [ <<string>> | <<url>> ] , <<custom-ident>> , <<counter-style>>? )
```
Comment (tabatkins) [inline css-content/Overview.bs]: Same as previous function.

File: css-content/Overview.bs [hunk 4]
```diff
@@ -851,7 +852,7 @@ The ''target-counters()'' function
 	and formats them by inserting a given string between the value of each nested counter.
 
 	<pre class="prod">
-		<dfn>target-counters()</dfn> = target-counter( [ <<string>> | <<url>> ] , <<custom-ident>> , <<string>> [ , <<counter-style>> ]? )
+		<dfn><<target-counters()>></dfn> = target-counters( [ <<string>> | <<url>> ] , <<custom-ident>> , <<string>> , <<counter-style>>? )
```
Comment (tabatkins) [inline css-content/Overview.bs]: Same as previous function.

File: css-content/Overview.bs [hunk 5]
```diff
@@ -871,7 +872,7 @@ The ''target-text()'' function</h4>
 	using the same values as the 'string-set' property above.
 
 	<pre class="prod">
-		<dfn id="target-text-function">target-text()</dfn> = target-text( [ <<string>> | <<url>> ] [ , [ content | before | after | first-letter] ]? )
+		<dfn id="target-text-function"><<target-text()>></dfn> = target-text( [ <<string>> | <<url>> ] , [ content | before | after | first-letter ]? )
```
Comment (tabatkins) [inline css-content/Overview.bs]: Same.

File: css-content/Overview.bs [hunk 6]
```diff
@@ -988,7 +989,7 @@ The string-set property</h4>
 The ''string()'' function</h4>
 
 	<pre class="prod">
-		<dfn>string()</dfn> = string( <<custom-ident>> , [ first | start | last | first-except ]? )
+		<dfn><<string()>></dfn> = string( <<custom-ident>> , [ first | start | last | first-except ]? )
```
Comment (tabatkins) [inline css-content/Overview.bs]: same

File: css-content/Overview.bs [hunk 7]
```diff
@@ -1087,7 +1088,7 @@ The ''string()'' function</h4>
 The ''content()'' function</h4>
 
 	<pre class="prod">
-		<dfn>content()</dfn> = content( [text | before | after | first-letter | marker ]? )
+		<dfn><<content()>></dfn> = content( [ text | before | after | first-letter | marker ] )
```
Comment (tabatkins) [inline css-content/Overview.bs]: same.

Also, don't remove the `?`, that's a significant and important bit. (It allows the "If no value is specified" bit a few lines later down.)

File: css-content/Overview.bs (hunk 1 above)
Comment (lahmatiy) [inline css-content/Overview.bs]: It's confusing a bit, but looks like I was wrong. Fixed.

File: css-content/Overview.bs (hunk 2 above)
Comment (lahmatiy) [inline css-content/Overview.bs]: `target` is not a function. I missed something?

File: css-content/Overview.bs (hunk 7 above)
Comment (lahmatiy) [inline css-content/Overview.bs]: Thank you for clarification, I missed this. Fixed.

File: css-content/Overview.bs (hunk 2 above)
Comment (tabatkins) [inline css-content/Overview.bs]: Yeah, sorry, was overcorrecting.

**Pull request merged**